test/udev_mapping_test.rb

```diff
@@ -4,50 +4,21 @@
 
 require "bootloader/udev_mapping"
 
-xdescribe Bootloader::UdevMapping do
+describe Bootloader::UdevMapping do
   subject { Bootloader::UdevMapping }
   before do
-    # always invalidate cache to use new mocks
-    allow(subject.instance).to receive(:cache_valid?).and_return false
-    allow(Yast::Arch).to receive(:ppc).and_return(false)
-    allow(Yast::Storage).to receive(:GetContVolInfo).and_return(false)
+    # always call proper method
     allow(Bootloader::UdevMapping).to receive(:to_kernel_device).and_call_original
```

Comment:
Please, remind me why we have these `allow(X).to receive(:y).and_call_original` here. Is to "disable" some mocking coming from the test_helper? In that case, "always call proper method" is not clear and obvious enough to me.

Reply:
yes, exactly. test_helper mock it by default, so I want to be sure, here is called real method. I will udpate comment